src/platform/unix.rs

```diff
@@ -41,6 +48,13 @@ pub trait WindowBuilderExtUnix {
   /// Set this window as a transient dialog for `parent`
   /// <https://gtk-rs.org/gtk3-rs/stable/latest/docs/gdk/struct.Window.html#method.set_transient_for>
   fn with_transient_for(self, parent: gtk::ApplicationWindow) -> WindowBuilder;
+
+  /// Disable the ability to draw transparent window automatically.
+  ///
+  /// When tranparent attribute is enabled, we will call `connect_draw` and draw the background automatically.
+  /// For anyone who wants to draw the background themselves, set this to `false`.
+  /// Default is `true`.
+  fn with_auto_transparent(self, auto: bool) -> WindowBuilder;
```

Comment:
I feel like this method name could be better, maybe something like this
```suggestion
  fn with_transparent_draw(self, draw: bool) -> WindowBuilder;
```